static/src/js/components/OrderProgressList/__tests__/OrderProgressItem.test.js

```diff
@@ -1,57 +1,121 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
 
-import { Badge, ProgressBar } from 'react-bootstrap'
+import { render, screen } from '@testing-library/react'
+
 import { OrderProgressItem } from '../OrderProgressItem'
 
-import { retrievalStatusPropsEsi } from '../../OrderStatus/__tests__/mocks'
+import {
+  retrievalStatusPropsEsi,
+  retrievalStatusPropsSwotOrder,
+  retrievalStatusPropsHarmonyOrder
+} from '../../OrderStatus/__tests__/mocks'
 
 beforeEach(() => {
   jest.clearAllMocks()
 })
 
-Enzyme.configure({ adapter: new Adapter() })
+const setup = (type, overrideProps) => {
+  let orderOrigin = null
+  let props = {}
+
+  switch (type) {
```

Comment:
A switch statement feels overly complicated here. There are two different times orderOrigin gets set to the ESI order which is difficult to track down the reason. And after setting orderOrigin you just pull the order out to pass in as props. 

Instead of writing this logic, just pass in the order to want to set as props for each test